pkg/3scale/amp/operator/base_apimanager_logic_reconciler.go

```diff
@@ -90,11 +92,16 @@ func (r *BaseAPIManagerLogicReconciler) ReconcileGrafanaDashboard(desired *grafa
 		return err
 	}
 
-	if !kindExists {
-		r.Logger().Info("Install grafana-operator in your cluster to create grafanadashboards objects", "Error creating grafanadashboard object", desired.Name)
+	if !r.apiManager.IsMonitoringEnabled() && !kindExists {
```

Comment:
What I would do and, IMO, much easier to understand:

```
if !kindExists {
    if r.apiManager.IsMonitoringEnabled() {
              r.Logger().Info("Install grafana-operator in your cluster to create grafanadashboards objects", "Error reconciling grafanadashboard object", desired.Name)
   
    }
    return nil
}
```

Reply:
I would also consider about creating namespace event for more visibility

Reply:
I thought about that other possibility yes.

Implementation is easier to understand depending on what we do with what's commented in https://github.com/3scale/3scale-operator/pull/509#discussion_r515076719.

:+1: about the event.

Reply:
Refactored with this approach :+1: